spatialconnect/src/main/java/com/boundlessgeo/spatialconnect/jsbridge/WebViewJavascriptBridge.java

```diff
@@ -39,6 +42,8 @@
     public WebViewJavascriptBridge(Activity context, WebView webview, WVJBHandler handler) {
         this.mContext = context;
         this.mWebView = webview;
+        // pass this instance to the bridge so it can call the appropriate handlers
+        ((SCJavascriptBridgeHandler) handler).setBridge(this);
```

Comment:
This isn't pretty but it gets the job done.  I think eventually we may need to refactor the WebViewJavascriptBridge to use Rx instead of keeping this workaround.